Workflow to create draft .pdf of the paper

speckhard opened the pull request:
JOSS submission guidelines recommend adding this action to the github to compile the paper https://github.com/marketplace/actions/open-journals-pdf-generator

Worked for me in testing in another private repo. Only issue I saw was the justification of some text but that was the fault of the action which I didn't write and is recommended by JOSS.

AlexBuccheri reviewed (approved):
I assume pandoc is implicit in one of the steps

AlexBuccheri commented:
If this is an action, why is there no sign of it running when you push?

speckhard commented:
Hmm I'm not sure why it didn't run an action:

I added you as a collaborator to a test github private repo I made where I added the same yml file and after merging it ran the action.
https://github.com/speckhard/excitingtools_test1

Let's see if it doesn't run now when we merge the new branch to main.
